File: API/models/Soilhumidity.js
const mongoose = require('mongoose');

const SoilHumiditySchema = mongoose.Schema({
    humidity: {
        type: String,
        required: true
    },
    date: {
        type: Date,
        required: true,
        default: Date.now
    },
    plantId: {
        type: String,
        required: true
    }
});

module.exports = mongoose.model('Soilhumidity', SoilHumiditySchema, 'soilhumidities');

File: App/PlantApp/PlantApp/Model/SoilHumidity.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PlantApp.Model
{
    class SoilHumidity
    {
        private int soilId;
        private string humidity;
        private DateTime date;

        public int SoilId { get => soilId; set => soilId = value; }
        public string Humidity { get => humidity; set => humidity = value; }
        public DateTime Date { get => date; set => date = value; }
    }
}
